### ISSUE TITLE
isDetox mock should probably be async

### ISSUE THREAD

**Issue by brianjuhl** [jesperjohansson/react-native-is-detox#1]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-is-detox@1.2.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-is-detox/jestSetup.js b/node_modules/react-native-is-detox/jestSetup.js
index 1b81728..ad12a79 100644
--- a/node_modules/react-native-is-detox/jestSetup.js
+++ b/node_modules/react-native-is-detox/jestSetup.js
@@ -1,4 +1,4 @@
 jest.mock('react-native-is-detox', () => ({
-  isDetox: () => false,
+  isDetox: async () => false,
   isDetoxSync: () => false,
 }));
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by jesperjohansson** [comment]
Fix published in v1.2.1 https://github.com/jesperjohansson/react-native-is-detox/releases/tag/v1.2.1

**Comment by brianjuhl** [comment]
Awesome! Thanks so much!
